library/choose_disk.py

```diff
@@ -361,6 +363,17 @@ def disk_label(partition):
     if "ceph journal" in stdout:
         return "journal"
 
+    # Then, let's search for metadata coming from ceph-volume
+    output_cmd = subprocess.Popen("ceph-volume lvm list --format=json {}".format(partition),
+                                  shell=True, stdout=subprocess.PIPE)
+    raw_json, _ = output_cmd.communicate()
+    json = json.loads(raw_json)
```

Comment:
this is blindly trusting the output of the CLI. If the CLI breaks, it will do so with invalid JSON, this must be caught and dealt with, even if it means it must fail here.

Reply:
Thx. Fixed.